yarn-project/end-to-end/src/e2e_event_logs.test.ts

```diff
@@ -57,7 +57,10 @@ describe('Logs', () => {
 
       const decryptedEvent0 = L1EventPayload.decryptAsIncoming(
         encryptedLogs[0],
-        deriveMasterIncomingViewingSecretKey(wallets[0].getSecretKey()),
+        computeAddressSecret(
+          wallets[0].getCompleteAddress().getPreaddress(),
+          deriveMasterIncomingViewingSecretKey(wallets[0].getSecretKey()),
+        ),
```

Comment:
Ideally we'd just return this from the wallet in one go, it makes no sense to fetch the components to then produce the secret whenever we need it.